save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/OrganizationView.kt

```diff
@@ -239,18 +256,46 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
                                 }
                                 +"Description"
                             }
-                            button(classes = "btn btn-link text-xs text-muted text-left ml-auto") {
-                                +"Edit  "
-                                fontAwesomeIcon(icon = faEdit)
-                                attrs.onClickFunction = {
-
+                            if ((state.roleInOrganization?.priority ?: 0) >= 2) {
+                                button(classes = "btn btn-link text-xs text-muted text-left ml-auto") {
+                                    +"Edit  "
+                                    fontAwesomeIcon(icon = faEdit)
+                                    attrs.onClickFunction = {
+                                        turnEditMode(isOff = false)
```

Comment:
on click you set `isEditDisabled = false`? Why?

Reply:
Don't hide. Allow field to be edited after clicking